.github/workflows/ci.yml

```
@@ -38,7 +36,7 @@ jobs:
           # https://github.com/aio-libs/yarl/issues/680
           YARL_NO_EXTENSIONS: 1
       - name: Run Tests
-        run: pytest --cov --cov-report=term --cov-report=xml
+        run: tox
```

And to match the `tox.ini` listing the versions to test, I usually use this to run just for the current Python version (e.g. [here](https://github.com/hugovk/pepotron/blob/e9d0511d0e87f5c0d506dbeb7614ffc9f2e06f33/.github/workflows/test.yml#L40) and [here](https://github.com/hugovk/pepotron/runs/6949910350?check_suite_focus=true#step:5:16)):

```suggestion
        run: tox -e py
```

---

Fixed in be31e3a.